src/PlusServer/vtkPlusOpenIGTLinkServer.cxx

```diff
@@ -1137,8 +1137,12 @@ void vtkPlusOpenIGTLinkServer::DisconnectClient(int clientId)
           }
           else
           {
-            // stop thread
+            // thread stopped
+            // We still need to call vtkMultiThreader::TerminateThread, not to terminate the thread (as it is already stopped)

```

Comment:
I suppose saying the thread is already stopped here is half-true, as the logic in vtkPlusOpenIGTLinkServer::DataReceiverThread has finished, but at this point we haven't called CloseHandle/Join on the thread. vtkMultiThreader::TerminateThread ends up calling CloseHandle/Join. I'm not sure if it's necessary to close the handle explicitly once the thread logic has completed / to mention this in the comment?